assets/js/data/payment/selectors.ts

```diff
@@ -73,8 +100,21 @@ export const expressPaymentMethodsInitialized = (
 	return state.expressPaymentMethodsInitialized;
 };
 
+/**
+ * @deprecated - use these selectors instead: isPaymentPristine, isPaymentStarted, isPaymentProcessing,
```

Comment:
Do you know about the `deprecated` function from `@wordpress/deprecated`? I think we should use that here.

Reply:
I did not, thanks for pointing it out, I've added it in now

Reply:
I didn't thanks for pointing it out, I've changed it now